docs/data-sources/aviatrix_all_transit_gateways.md

```diff
@@ -0,0 +1,60 @@
+---
+subcategory: "Multi-Cloud Transit"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_all_transit_gateways"
+description: |-
+Gets a list of all Aviatrix transit gateway's details.
+---
+
+
+# aviatrix_all_transit_gateways
+
+The **aviatrix_all_transit_gateways** data source provides details about all transit gateways created by the Aviatrix Controller.
+
+## Example Usage
+
+```hcl
+# Aviatrix All Transit Gateways Data Source
+data "aviatrix_all_transit_gateways" "foo" {
```

Comment:
Since this data source has no input attributes, we probably don't need the extra newlines e.g. `data "aviatrix_all_transit_gateways" "foo" {}

Reply:
have fixed